.gitignore

```diff
@@ -3,3 +3,5 @@
 .idea/
 node_modules
 coverage
+dist/README.md
```

Comment:
The thing this makes me thing is should the whole dist directory be in the gitignore, and generated by the build? On first look I was a bit surprised we had the dist files in here already, but there might be a reason that I don't know about being a noob in js :-)

Reply:
Thanks Becka - I’m not super experienced with publishing NPM packages so all suggestions are super welcome 👍🏽 

Initially, I was thinking we need the `/dist` directory in source control because it prevents consumers from having to build it. That initially felt like a fairly common pattern for NPM packages and what teams that are more experienced with publishing packages in GDS do (for example, [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) includes the `/dist` directory).

Reflecting on what you’ve asked, I realise that the `/dist` directory doesn’t necessarily need to be version controlled to be published in a way that consumers don’t need to build it. 

I’ve added a new commit which does this.